blazor/diagram/serialization.md

```diff
@@ -41,6 +41,25 @@ string data = Diagram.SaveDiagram();
 await Diagram.LoadDiagram(data);
 ```
 
+## Load the SfDiagramComponent from SfDiagram string
+
+The [LoadDiagram](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.Diagram.SfDiagramComponent.html#Syncfusion_Blazor_Diagram_SfDiagramComponent_LoadDiagram_System_String_) method loads the [SfDiagramComponent](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.Diagram.SfDiagramComponent.html) from the serialized text data of the [SfDiagram](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.Diagrams.SfDiagram.html).
```

Comment:
You can load the SfDiagram serialized JSON data string into SfDiagramComponent using Load method. When you load SfDiagram serialized string, then the isClassicData parameter should be set to true. The default value of the isClassicData is false.